RadarSDK/Include/RadarLocationPermissionStatus.h

```diff
@@ -20,9 +20,11 @@
 typedef NS_ENUM(NSInteger, RadarLocationPermissionState) {
     NoPermissionGranted,
     ForegroundPermissionGranted,
+    ApproximateForegroundPermissionGranted,
     ForegroundPermissionRejected,
     ForegroundPermissionPending,
     BackgroundPermissionGranted,
+    ApproximateBackgroundPermissionGranted,
```

Comment:
has these been added by apple? if not, then we should leave them out even if android has them.